src/bindings/python/flux/kvs.py

```diff
@@ -116,7 +116,7 @@ def put_symlink(flux_handle, key, target):
 
 def commit(flux_handle, flags: int = 0) -> int:
     if flux_handle.aux_txn is None:
```

Comment:
I struggled a bit with this since calling `commit()` with no transaction felt more like calling `flux_kvs_commit()` with a `NULL` transaction, which is an error. But because of the current practice of one-transaction-per-handle I think your approach here is correct.

However, it isn't very idiomatic Python to return 0 from a function to test for success. Should the function just return nothing and raise an exception on error? (Apologies if you fix this in a later commit)